spec/requests/users_spec.rb

```diff
@@ -2,6 +2,18 @@
 
 RSpec.describe 'Users', type: :request do
```

Comment:
- [ ] The index and show actions for the user controller should be tested as requested in the requirements. They should be a `describe` block for both `index` and `show` actions for best practices. **Hints**


```ruby
describe 'GET /index' do
   it('status should be :ok') do
      get '/users/index'
      expect(response).to have_http_status(:ok)
    end

    it('rendered :index template') do
      get '/users/index'
      expect(response).to render_template(:index)
    end
    it 'body should includes correct placeholder text' do
      get '/users/index'
      expect(response.body).to include('Here is all users')
    end
end

describe 'GET /show' do
  ...
end
```

- kindly do the same for GET/show request